lib/jsonld_serializers.js

```diff
@@ -109,6 +109,10 @@ class JsonLdItemSerializer extends JsonLdSerializer {
       }
     })
 
+    // Add hasVol and hasDate properties
+    stmts.hasItemVolumes = this.body.numItemVolumesParsed / this.body.numItems > process.env.BIB_HAS_VOLUMES_THRESHOLD
```

Comment:
Use `parseFloat` on the env var just to be safe. (Otherwise you're coercing a string comparison, which I don't think changes much for these values, but let's be certain it's a numeric comparison)